server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -505,22 +507,45 @@ export class SummaryWriter implements ISummaryWriter {
     private async generateLogtailEntries(
         from: number,
         to: number,
-        pending: ISequencedOperationMessage[]): Promise<ITreeEntry[]> {
+        pending: ISequencedOperationMessage[],
+        lastSummaryMessages: ISequencedDocumentMessage[] | undefined): Promise<ITreeEntry[]> {
         const logTail = await this.getLogTail(from, to, pending);
+
+        // Some ops would be missing if we switch cluster during routing.
+        // We need to load these mssing ops from the last summary.
+        const missingOps = await this.getSummaryOps(from, to, logTail, lastSummaryMessages);
+        const fullLogTail = missingOps ? (missingOps.concat(logTail)).sort((ms) => ms.sequenceNumber) : logTail;
```

Comment:
It will be useful to log if there are any missing ops in fullLogTail.

Reply:
Got it. If we have missing ops in the last summary, we would log it in the getSummaryOps method.

Reply:
Got it. Added "Fetched the missing ops from the last summary"

Reply:
Done. :)